text/0000-label-rfcs.md

```diff
@@ -0,0 +1,35 @@
+# Meta
+[meta]: #meta
+- Name: Label RFCs With Specification And Target Audience
+- Start Date: 2020-08-15
+- Author(s): ForestEckhardt
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (N/A)
+
+# Summary
+[summary]: #summary
+
+It would be nice to start labeling RFCs with a label that indicates which part
+of the specification (i.e. Buildpack Interface, Distribution, Platform
+Interface) is being affected as well as a label to indicate the target audience
+(i.e. Buildpack Authors, CNB Contributors, Platform Operators) of the change.
+
+# Motivation
+[motivation]: #motivation
+
+Indicating which part of the specification the RFC is purposing to update or
+modify will make the RFC more discoverable because it will allow people to
+focus in on certain RFCs that effect them. This means that potentially effected
+parties have to spend less time wading through all of the RFCs in the
+repository to find the handful that will affect them. Overall, this change will
+make reviewing RFCs a much more manageable process for new members and people
+that are concerned with specific parts of the specification.
+
+# Implementation
+[Implementation]: #implementation
+
+During any Working Group meeting, when a new RFC is brought up it should at
```

Comment:
Should we try to do this during the weekly review of RFCs or would that bog that process down too much? Thoughts @sclevine?

Reply:
From the WG: we should be able to handle this as we go through the RFCs every week if you want to update @ForestEckhardt

Reply:
@hone Sounds good to me. I will update this when I get the change.